mockintosh/handlers.py

```diff
@@ -50,11 +51,35 @@
 
 class GenericHandler(tornado.web.RequestHandler):
 
-    def initialize(self, config_dir, endpoints, _globals, definition_engine, interceptors):
+    def prepare(self):
+        self.dont_add_status_code = False
+        self.special_request_start_time = time.perf_counter()
+        super().prepare()
+
+    def on_finish(self):
+        if not self.__class__.__name__ == 'ErrorHandler' and not self.is_options and self.methods is not None:
+            if self.get_status() != 405:
+                self.set_elapsed_time()
+            if not self.dont_add_status_code:
+                self.stats.services[self.service_id].endpoints[self.internal_endpoint_id].add_status_code(
+                    str(self.get_status())
+                )
+        super().on_finish()
+
+    def set_elapsed_time(self):
+        elapsed_time_in_microseconds = time.perf_counter() - self.special_request_start_time
```

Comment:
Is it still microseconds? Or seconds?

Reply:
Fixed with https://github.com/up9inc/mockintosh/commit/4c2b1a3346cf9c803eae8aec1cad950efb2b30f0